sysmacros patch generated by KarLS Builder Infrastructure
sysmacros.h not defined
diff -ru --no-dereference src/syslinux-6.03.orig/syslinux-6.03/extlinux/main.c src/syslinux-6.03/extlinux/main.c
--- src/syslinux-6.03.orig/syslinux-6.03/extlinux/main.c	2014-10-06 18:27:44.000000000 +0200
+++ src/syslinux-6.03/extlinux/main.c	2021-04-30 17:26:42.833063693 +0200
@@ -39,6 +39,7 @@
 #include <sys/ioctl.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <sys/mount.h>
 #include <sys/vfs.h>
 
